From: Snild Dolkow <snild@sony.com>
Date: Tue, 26 Sep 2023 10:08:58 +0200
Subject: tests: set isFinal in test_line_number_after_parse

Without this, parsing of the start or end tag may be deferred, yielding
an unexpected line number.

Origin: https://github.com/libexpat/libexpat/commit/2e1253414559d2649cbf5662496800061034eb49
---
 expat/tests/runtests.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/expat/tests/runtests.c b/expat/tests/runtests.c
index 47a5b15..9bdf716 100644
--- a/expat/tests/runtests.c
+++ b/expat/tests/runtests.c
@@ -1067,7 +1067,7 @@ START_TEST(test_line_number_after_parse) {
                      "\n</tag>";
   XML_Size lineno;
 
-  if (_XML_Parse_SINGLE_BYTES(g_parser, text, (int)strlen(text), XML_FALSE)
+  if (_XML_Parse_SINGLE_BYTES(g_parser, text, (int)strlen(text), XML_TRUE)
       == XML_STATUS_ERROR)
     xml_failure(g_parser);
   lineno = XML_GetCurrentLineNumber(g_parser);
